Gemfile

```diff
@@ -1,8 +1,15 @@
 # frozen_string_literal: true
 
-source "https://rubygems.org"
+source 'https://rubygems.org'
 
 # Specify your gem's dependencies in globus_client.gemspec
 gemspec
 
-gem "byebug"
+group :deployment do
```

Comment:
Rubocop suggests development dependencies should not be in the gemspec, but rather in the development group of the Gemfile